roles/setup_repmgr/tasks/repmgr_configure_spl.yml

```diff
@@ -39,10 +39,32 @@
   when:
     - update_spl
 
+- name: Update systemd unit file
+  ansible.builtin.lineinfile:
+    path: "/etc/systemd/system/{{ pg_service }}.service"
+    line: "{{ item.line }}"
+    regexp: "{{ item.regexp }}"
+    insertafter: "{{ item.insertafter | default('EOF') }}"
+  become: true
+  loop:
+    - line: "Description=Database server {{ pg_type }} {{ pg_version }} - Instance: {{ pg_instance_name }}"
+      regexp: "^Description=.*"
+    - line: "Environment=PGDATA={{ pg_default_data }}"
+      regexp: "^Environment=PGDATA.*"
+    - line: "PIDFile={{ pg_default_data }}/postmaster.pid"
+      regexp: "^PIDFile=.*"
+      insertafter: "^\\[Service\\]$"
+    - line: "ExecStopPost=+/usr/bin/systemctl daemon-reload"
+      regexp: "^ExecStopPost=.*"
+      insertafter: "^\\[Service\\]$"
+  when:
+    - ansible_distribution in ['RedHat', 'CentOS', 'Rocky']
+
```

Comment:
I'm not in favor of executing this task this way and at this place. In theory, you shouldn't have to update this systemd file, please take a look at https://github.com/EnterpriseDB/edb-ansible/blob/master/roles/setup_repmgr/tasks/repmgr_setup_systemd.yml#L8-L16 and https://github.com/EnterpriseDB/edb-ansible/blob/master/roles/setup_repmgr/templates/override.conf.j2